samples/bluetooth/direct_test_mode/src/fem/generic_fem.c

```diff
@@ -86,6 +86,14 @@ static int gpio_config(void)
 	}
 #endif /* DT_NODE_HAS_PROP(DT_NODELABEL(nrf_radio_fem), chl_gpios) */
 
+#ifdef __cplusplus
```

Comment:
Wouldn't be a simpler solution to add the __used attribute to this structure definition?

Reply:
Actually it would be simpler